src/components/DraftImportCsvModal.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal"> 
+          <ion-icon slot="icon-only" :icon="closeOutline" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>{{ translate("Import CSV") }}</ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content>
+    <ion-item>
+      <ion-select :label="translate('Product identifier')" interface="popover" placeholder="Select">
+        <ion-select-option>example-1</ion-select-option>
+        <ion-select-option>example-2</ion-select-option>
+        <ion-select-option>example-3</ion-select-option>
+      </ion-select>
+    </ion-item>
+    
+    <ion-list>
+      <ion-list-header>{{ translate("Select the column containing products") }}</ion-list-header>
+      <ion-item>
+        <ion-label>column 1</ion-label>
+        <ion-radio aria-label="col" slot="end"></ion-radio>
```

Comment:
```suggestion
        <ion-radio aria-label="col" slot="end">column 1</ion-radio>
```

Update this on all the places and wrap ion-radio in radio-group as we only need to select a single value out of all the options available.